grunt/config/javascript.js

```diff
@@ -18,6 +18,19 @@ module.exports = function(grunt, options) {
         pluginsFilter: function(filepath) {
           return grunt.config('helpers').includedFilter(filepath);
         },
+        umdImports: [
+          '../node_modules/object.assign/dist/browser.js',
+          '../node_modules/react/umd/react.production.min.js',
+          '../node_modules/react-dom/umd/react-dom.production.min.js',
+          '../node_modules/html-react-parser/dist/html-react-parser.min.js'
+        ],
+        reactTemplates: [
+          '<%= sourcedir %>core/templates/**/*.jsx',
+          '<%= sourcedir %>components/*/templates/**/*.jsx',
+          '<%= sourcedir %>extensions/*/templates/**/*.jsx',
+          '<%= sourcedir %>menu/*/templates/**/*.jsx',
+          '<%= sourcedir %>theme/*/templates/**/*.jsx'
```

Comment:
Just checking, should these be conditionally set for the plugins on use on the course rather than a wildcard?

Reply:
That is a good find. Will sort. Ta.